### DISCUSSION TITLE
Record plugin closes mic with stopMic function but chrome still shows recording

### DISCUSSION THREAD

**Discussion by e-senturk** [katspaugh/wavesurfer.js#3248] [Q&A] [answered]
Hi, I am using Wavesurfer with ReactJS and Record plugin it works well and record-start and record-end are setting and triggering only once in my code. recorder.stopMic(); is working on my side and the waveform is no longer generated. But even if I closed the mic it shows used for the chrome tab. Can I disable it when it's not recording voice? I tried calling destroy, stop recording, and unAll methods in the record-end callback, but none worked.

recorder.on('record-start', () => {
        recorder.startMic();
        setRecordingDuration(0);
        const id = setInterval(() => {
          setRecordingDuration((prevDuration) => prevDuration + 1);
        }, 1000);
        setIntervalId(id);
      });
      recorder.on('record-end', (blob) => {
        recorder.stopMic();
        clearInterval(intervalId);
        const recordedUrl = URL.createObjectURL(blob);
        props?.setRecordUrl?.(recordedUrl);
      });

**Comment by AmitJoki** [comment, marked as answer]
@katspaugh this is because the MediaStream is still alive even after media recorder is stopped. We have to stop the individual tracks. I have fixed it this in the commit https://github.com/katspaugh/wavesurfer.js/pull/3249/commits/f58b734215b461e936d0e5397f3ed6aa3d33f97f in the PR https://github.com/katspaugh/wavesurfer.js/pull/3249
